tests/ui/test_ui.py

```diff
@@ -58,6 +59,14 @@ def test_set_footer_text_default(self, view, mocker):
         view._w.footer.set_text.assert_called_once_with(['some help text'])
         view.controller.update_screen.assert_called_once_with()
 
+    def test_set_footer_text_with_duration(self, view, mocker):
+        mocker.patch('zulipterminal.ui.View.get_random_help',
+                     return_value=['some help text'])
+        view.set_footer_text("custom text", 0.1)
+        view._w.footer.set_text.assert_called_with(['some help text'])
```

Comment:
Minor: maybe extract the common text as a test function parameter?

Reply:
I think it's fine since we are not testing it here.

Reply:
`'some help text'` ?

Reply:
`test_set_footer_text_default` is test for that, right?

Reply:
I agree that the other test uses that text; I think the issue is which combination of parameters are used? This test sets a duration with a random help test? This doesn't actually correspond to how you're then using the code, which is with specific text and a duration?